python/hail/__init__.py

```diff
@@ -1,21 +1,43 @@
-from hail.representation import IntervalTree
-
 import hail.expr
+from hail.representation import *
 from hail.context import HailContext
 from hail.dataset import VariantDataset
-from hail.expr import Type
+from hail.expr import *
 from hail.keytable import KeyTable
 from hail.kinshipMatrix import KinshipMatrix
-from hail.utils import TextTableConfig, hadoop_read, hadoop_write, hadoop_copy
+from hail.utils import hadoop_read, hadoop_write, hadoop_copy
 
 __all__ = ['HailContext',
            'VariantDataset',
            'KeyTable',
-           'TextTableConfig',
-           'IntervalTree',
-           'expr',
-           'representation',
+           'Variant',
+           'Locus',
+           'AltAllele',
+           'Interval',
+           'Genotype',
+           'Struct',
+           'Call',
+           'Pedigree',
+           'Trio',
+           'Type',
+           'TInt',
+           'TLong',
+           'TFloat',
+           'TDouble',
+           'TString',
+           'TBoolean',
+           'TArray',
+           'TSet',
+           'TDict',
+           'TStruct',
+           'TLocus',
+           'TVariant',
+           'TAltAllele',
+           'TGenotype',
+           'TCall',
+           'TInterval',
            'hadoop_read',
            'hadoop_write',
            'hadoop_copy',
-           'KinshipMatrix']
```

Comment:
Good catch. Not sure this really needs to be here, though, since the only constructor is the java constructor and there are no static methods